build-system/tasks/presubmit-checks.js

```diff
@@ -164,8 +164,8 @@ var forbiddenTerms = {
   'installStorageService': {
     message: privateServiceFactory,
     whitelist: [
-      'extensions/amp-analytics/0.1/amp-analytics.js',
-      'extensions/amp-analytics/0.1/storage-impl.js',
+      'src/runtime.js',
```

Comment:
Yeah I think we can drop this presubmit check from here. @jridgewell do you know?